download.sh

```diff
@@ -27,14 +27,26 @@ rm /opt/gpu/${RUNFILE}.run
 popd
 
 
+install_fabric_manager () {
+    local driver_ver="${DRIVER_VERSION}"
+    if [[ "${DRIVER_VERSION}" == "535.161.08" ]]; then
+      local driver_ver="535.104.05"
```

Comment:
maybe rename `fabric_manager_version`

Reply:
not sure you need local here, since it's been declared outside of the if, you are simply looking to reassign a new value right ?